charts/pelorus/charts/exporters/templates/_deploymentconfig.yaml

```diff
@@ -51,6 +52,16 @@ spec:
         ports:
         - containerPort: 8080
           protocol: TCP
+        readinessProbe:
+          tcpSocket:
+            port: 8081
+          initialDelaySeconds: 5
+          periodSeconds: 10
+        livenessProbe:
+          tcpSocket:
+            port: 8080
+          initialDelaySeconds: 15
+          periodSeconds: 20
```

Comment:
would threshholds be useful here?
successThreshold: 1 
failureThreshold: 3 
https://docs.openshift.com/container-platform/4.10/applications/application-health.html

Reply:
By default it's 1 and 3, so I think we are fine here. If exporter is not exposing 8080 on localhost, then we are in a trouble.

There is benefit of having those probes as this will kill pods and remove from the cluster to not occupy resources. There is also a drawback. Once the pod is killed it's gone and we can't simply get the logs from such pod to see what happened within python application, however we can always restart deployment and while the pod is running investigate the issue.